skimage/draw/draw.py

```diff
@@ -693,3 +693,57 @@ def bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape=None):
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
     """
     return _bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape)
+
+
+def rectangle(origin, extent, shape=None):
+    """Generate coordinates of pixels within an nd-rectangle.
+
+    Parameters
+    ----------
+    origin : tuple
+        origin of the rectangle, (page,row,column,channel)
+    extent : tuple
+        extent (num_pages,num_rows,num_cols,num_chanels..) of the rectangle.
```

Comment:
Hi jni,
Thanks for the feedback. I guess I would argue for the origin, extent signature since it is the same as the cv2 and matplotlib calls. Plus, it seems that really the utility of the function is to save the user the trouble of having to clutter their code with some simple math.  Perhaps it would be useful to have a general function that finds the rectilinear volume enclosing a set of points. This would work with the two point call you are suggesting as well as a polygon. If it is useful, one might also find an enclosing ellipse.